docs/cram-tests/index.md

```diff
@@ -22,14 +22,6 @@ which means:
 - You can run all tests using a single command that doesn't need to refer to
   individual tests
 
-Before you can write your first cram test, you need to enable cram tests by
-adding this to your `dune-project` file:
-
-```dune
-; Enable cram tests
-(cram enable)
```

Comment:
Is this mentioned somewhere else?

Reply:
It's in the issue related to this PR: https://github.com/melange-re/melange-for-react-devs/issues/35

I've added a link to the issue in the description, should've done so from the start.